components/Chips/src/MDCChipView.m

```diff
@@ -510,15 +510,15 @@ - (void)updateTitleFont {
 
   // If we are automatically adjusting for Dynamic Type resize the font based on the text style
   if (self.mdc_adjustsFontForContentSizeCategory) {
-    if (titleFont.mdc_scalingCurve && !self.mdc_legacyFontScaling) {
```

Comment:
Since this is deprecated and not removed, you cannot yet remove the existing logic.

Reply:
Addressed in 482d459770672b9512ec79eecdc8e615c66e4131.